packages/nextjs/generated/deployedContracts.ts

```diff
@@ -1,3 +1,15 @@
-const deployedContracts = null;
+import { GenericContractsDeclaration } from "~~/utils/scaffold-eth/contract";
+
+/**
+ * @example
+ * const deployedContracts = {
+ *   31337: {
+ *    YourContract: {
+ *      address: "0x..."
+ *      abi: [...],
+ *    }
+ * } satisfies GenericContractsDeclaration | null;
+ */
```

Comment:
What about adding something like:

```ts
/**
 * This file is populated when deploying contracts from Scaffold-ETH 2.
 * You can also add external (or existing??) with the following structure
 * .....
 */
``` 

About the example, maybe we could use a "real example"

```
/**ts
 * @example
 * const deployedContracts = {
 *  // chainId
 *   1: {
 *    // Contract name
 *    DAI: {
 *      address: "0x..."
 *      abi: [...],
 *    }
 * };
 */
```

IDK, just thinking out loud!